org.gramps_project.Gramps.yml

```diff
@@ -0,0 +1,87 @@
+app-id: org.gramps_project.Gramps
+# flatpak-builder will not take com.gramps-project.Gramps
+runtime: org.gnome.Platform
+runtime-version: '3.36'
```

Comment:
Update runtime to 3.38

Reply:
Are you sure 3.38 is available for the SDK?  I'm getting an error that says it can't find the 3.38 SDK.
```
error: org.gnome.Sdk/x86_64/3.38 not installed
Failed to init: Unable to find sdk org.gnome.Sdk version 3.38
Error: Process completed with exit code 1.
```